source/playbooks/NIST80053/ssmdocs/scripts/check_for_s3_bucket_name.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this duplicated? If it's needed here as well as in SC, it needs to be moved to the "common" scripts location so it is not duplicated. Multiple instances.